spec/controllers/miq_task_controller_spec.rb

```diff
@@ -851,6 +854,17 @@
       end
     end
 
+    describe "#list_jobs" do
+      it 'sets the active tab' do
+        controller.instance_variable_set(:@tabform, "ui_2")
```

Comment:
yea, this is a side effect of all of those `@` variables vs passing in the variables into `list_jobs`